Source/Views/StackViewController.swift

```diff
@@ -9,6 +9,10 @@
 import UIKit
 import SnapKit
 
+public enum RelativePosition {
+    case topOrLeft, center, bottomOrRight
```

Comment:
split case in multiple lines?
BTW instead of topOrLeft / bottomOrRight maybe beginning end? not sure